src/pages/individual/Verify.jsx

```diff
@@ -5,7 +5,7 @@ import { FaArrowLeft } from "react-icons/fa";
 
 const Verify = () => {
   const navigate = useNavigate();
-  const { currentStep } = useOutletContext();
+  const { currentStep: _currentStep } = useOutletContext();
```

Comment:
I do not think this change is needed. If `currentStep` is an unused var then remove entirely